app/javascript/components/App.js

```diff
@@ -44,7 +67,12 @@ class App extends Component {
             <Route path="/apartmentindex" render = {(props) => <ApartmentIndex {...props} apartments={this.state.apartments} />} />
             <Route path="/apartmentshow" component={ApartmentShow} />
             <Route path="/apartmentnew" component={ApartmentNew} />
-            <Route path="/apartmentedit" component={ApartmentEdit} />
+            <Route path="/apartmentedit" render = {(props) => {
+              const { match } = props
+              const { id } = match.params
+              return <ApartmentEdit {...props} apartment={this.state.apartments.find(apartment => apartment.id === id)} />
```

Comment:
Is this showing the page correctly?  What datatype is id coming from match.params?

Reply:
So I realized that I didnt have :id in the path. I went back and looked at cat tinder and commented out this code to see if the cat tinder version would be better.